dapp/client/App.elm

```diff
@@ -16,13 +16,16 @@ type alias Amount =
 
 type alias Model =
     { address : Maybe Address
-    , eth : Amount
+    , wallet : Maybe Address
+    , eth : Maybe Amount
```

Comment:
All of these properties start out as Nothing, so they all need to be `Maybe` types.